pyproject.toml

```diff
@@ -1,11 +1,10 @@
 [tool.poetry]
 name = "babelon"
-version = "0.2.1"
+version = "0.0.0"
```

Comment:
Do we have to change the version? We already have some published, I want to start at 0.2. if possible

Reply:
No ....when you do a release via GitHub, it should get the version from the tag provided for release.